<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setup_control_MainWindow</class>
 <widget class="QMainWindow" name="setup_control_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qudi: Setup Control QINU</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Control</string>
        </property>
       </widget>
      </item>

      <item row="3" column="1">
       <widget class="QPushButton" name="Flipmirror_Button">
        <property name="text">
         <string>Powermeter in</string>
        </property>
       </widget>
      </item>

      <item row="2" column="1">
       <widget class="QPushButton" name="Lamp_Button">
        <property name="text">
         <string>Lamp</string>
        </property>
       </widget>
      </item>

      <item row="1" column="1">
       <widget class="QPushButton" name="Green_Button">
        <property name="text">
         <string>Green</string>
        </property>
        <property name="text">
         <string>Green</string>
        </property>
       </widget>
      </item>

      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Power</string>
        </property>
       </widget>
      </item>

      <item row="1" column="2">
       <widget class="QSlider" name="green_attenuation">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="minimum">
         <double>0</double>
        </property>
        <property name="maximum">
         <double>500</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>Lamp_Button</tabstop>
  <tabstop>Green_Button</tabstop>
  <tabstop>Flipmirror_Button</tabstop>
  <tabstop>green_attenuation</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
